cwms-data-api/src/main/java/cwms/cda/data/dao/BlobDao.java

```diff
@@ -158,14 +158,14 @@ public void update(Blob blob, boolean ignoreNulls) {
             throw new NotFoundException("Null blob provided to update");
         }
 
-        if (!blobExists(blob.getOfficeId(), blob.getId())) {
-            throw new NotFoundException("Unable to find blob with id " + blob.getId() + " in office " + blob.getOfficeId());
+        if (!blobExists(blob.getOfficeId(), blob.getId().toUpperCase())) {
```

Comment:
blob exists already does the upper, it makes sense to have blobExists do the upper.

Perhaps just add javadoc to blobExists that an upper case comparison will be done.